test/test_flow_control_window.py

```diff
@@ -52,6 +52,32 @@ def test_flow_control_decreases_with_sent_data(self):
         remaining_length = self.DEFAULT_FLOW_WINDOW - len(b'some data')
         assert (c.local_flow_control_window(1) == remaining_length)
 
+    def test_flow_control_decreases_with_sent_data_with_padding(self):
+        """
+        When padded data is sent on a stream, the flow control window should
+        drop by the length of the padding plus 1 for the 1-byte padding length
+        field.
```

Comment:
This is a minor nit, but we try to write these docstrings as imperative: that is, we don't write that X *should* happen, we write that it *does* happen. For example, this would be:

> When padded data is sent on a stream, the flow control window drops by the length of the padding plus 1 for the 1-byte padding length field.